extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/S2iConfig.java

```diff
@@ -15,6 +15,15 @@
     public static final String DEFAULT_BASE_NATIVE_IMAGE = "quay.io/quarkus/ubi-quarkus-native-binary-s2i:1.0";
     public static final String DEFAULT_NATIVE_TARGET_FILENAME = "application";
 
+    public static final String DEFAULT_JVM_DOCKERFILE = "src/main/docker/Dockerfile.jvm";
```

Comment:
I don't understand why we want to add to docker related stuff to S2iConfig.

I assume that this is because the `container-image-s2i` variant has them (which IMHO is wrong and I am to blame).

I would expect that this PullRequest would remove them from `container-image-s2i` instead of introducing them to `container-image-openshift` if that makes sense.

In the end of the day both `S2iConfig` should be the same but with different defaults (as we already chatted about).

Reply:
Yea I'm adding it to the new S2I config, because the old S2I config has it (and is presumably used).

Reply:
Also the new OpenShift container image config has these properties.

My assumption is that the old S2I (`quarkus-container-image-s2i`) is going away (soon?) and the new S2I config (`quarkus.s2i.*` config properties as provided by `quarkus-container-image-openshift`) is only there for migration. Ideally everyone would move to the OpenShift container image config (`quarkus.openshift.*` config properties as provided by `quarkus-container-image-openshift`).

Right?

Reply:
OK, just read the Zulip message -- clearly some config properties were added to the "old" S2I config recently by mistake. Indeed those should be removed, and I've just done that.